### PR TITLE
Feature kotlin extensions

### PR DESCRIPTION
**Pull request by JarnaChao09**
uwu, pwease mewge mwy pwuwww weqwest into mwastew, senpai~! OwO

fow uw pr, pwease make suwwe it answews the fowwowing:
## what kind of change does this pr intwoduwce? UwU
* stywish code uwwuw

adds syntactic sugars for Kotlin users of FTCLib

this does not add any breaking changes

### CODE REVIEW

**Comment by JarnaChao09** [comment]
Do not merge, will change `in` to input and output for all LUT related files

**Comment by JIceberg** [comment]
Converted to draft

**Comment by JIceberg** [comment]
Yo but did it work tho @JarnaChao09

File: core/src/main/java/com/arcrobotics/ftclib/kotlin/extensions/drivebase/RobotDriveExt.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.arcrobotics.ftclib.kotlin.extensions.drivebase
+
+import com.arcrobotics.ftclib.drivebase.RobotDrive
+
```
Comment (JIceberg) [inline core/src/main/java/com/arcrobotics/ftclib/kotlin/extensions/drivebase/RobotDriveExt.kt:4]: Required comment that explains what this class is doing for the extension.

File: core/src/main/java/com/arcrobotics/ftclib/kotlin/extensions/geometry/Pose2dExt.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package com.arcrobotics.ftclib.kotlin.extensions.geometry
+
+import com.arcrobotics.ftclib.geometry.Pose2d
+import com.arcrobotics.ftclib.geometry.Transform2d
+import com.arcrobotics.ftclib.geometry.Twist2d
+
```
Comment (JIceberg) [inline core/src/main/java/com/arcrobotics/ftclib/kotlin/extensions/geometry/Pose2dExt.kt:6]: Needs a comment that explains what this class is doing for the extension.

File: core/src/main/java/com/arcrobotics/ftclib/kotlin/extensions/geometry/Rotation2dExt.kt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package com.arcrobotics.ftclib.kotlin.extensions.geometry
+
+import com.arcrobotics.ftclib.geometry.Rotation2d
+
```
Comment (JIceberg) [inline core/src/main/java/com/arcrobotics/ftclib/kotlin/extensions/geometry/Rotation2dExt.kt:4]: Needs a comment that explains what this class is doing for the extension.

**Review by JIceberg — CHANGES_REQUESTED** [review]
Please add comments at the top of each class that explains what it is doing along with its purpose.

**Review by JIceberg — APPROVED** [review]
Looks good.

**Pull request merged**